pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -406,6 +406,24 @@ type MonitoringSpec struct {
 	// Interval determines prometheus scrape interval
 	// +optional
 	Interval *metav1.Duration `json:"interval,omitempty"`
+
+	// Ceph exporter configuration
+	// +optional
+	Exporter *CephExporterSpec `json:"exporter,omitempty"`
+}
+
+type CephExporterSpec struct {
+	// Only performance counters greater than or equal to this option are fetched
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=10
+	// +kubebuilder:default=5
+	PerfCountersPrioLimit int `json:"perfCountersPrioLimit,omitempty"`
+
+	// Time to wait before sending requests again to exporter server (seconds)
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=600
+	// +kubebuilder:default=5
+	StatsPeriod int `json:"statsPeriod,omitempty"`
```

Comment:
If it has "period" in the name, I would expect units to be allowed such as `30s` or `2m`, but if it's just seconds, it is more clear to name it as such.
```suggestion
	StatsDelaySeconds int `json:"statsDelaySeconds,omitempty"`
```